classes/exception/PrestaShopException.php

```diff
@@ -42,7 +42,7 @@ public function displayMessage()
         if (ToolsCore::isPHPCLI()) { 
             echo get_class($this).' in '. $this->getFile() .' line '. $this->getLine()."\n";
             echo $this->getTraceAsString()."\n";
-        } elseif (_PS_MODE_DEV_ || defined('_PS_ADMIN_DIR_')) {
+        } elseif (_PS_MODE_DEV_) {
```

Comment:
Why removing this check?

Reply:
because we don't want to display arguments and errors if we are in production and in admin directory.